spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-config-it/src/test/resources/spring-cloud-kubernetes-client-config-it-deployment.yaml

```diff
@@ -14,16 +14,16 @@ spec:
       serviceAccountName: spring-cloud-kubernetes-serviceaccount
       containers:
         - name: spring-cloud-kubernetes-client-config-it
-          image: localhost:5000/spring-cloud-kubernetes-client-config-it:2.0.0-SNAPSHOT
+          image: docker.io/springcloud/spring-cloud-kubernetes-client-config-it:2.0.0-SNAPSHOT
           imagePullPolicy: IfNotPresent
           readinessProbe:
             httpGet:
               port: 8080
-              path: /actuator/health/readiness
+              path: /actuator/health
```

Comment:
Why are you changing the readiness and liveness endpoints?

Reply:
reverted

Reply:
Doesnt look like it was reverted

Reply:
ah.. oversight.. plz take another look